backend/requirements.txt

```diff
@@ -100,6 +103,10 @@ webencodings==0.5.1       # via tinycss2
 webob==1.8.4              # via pyramid
 wsgidav==2.4.1
 xvfbwrapper==0.2.9        # via preview-generator
+zipp==0.6.0               # via importlib-metadata
 zope.deprecation==4.3.0   # via pyramid
 zope.interface==4.6.0     # via pyramid, pyramid-retry, transaction, zope.sqlalchemy
 zope.sqlalchemy==1.0
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools
```

Comment:
This is generated, but cant it be skipped easly ?

Reply:
Don't think so, it'seem that one of our dependencies require setuptools.

Reply:
Checked. Don't see way to disable this line. we can allow unsafe requirement but that's not a good idea.
Just done it to know which dependencies require setuptools:
```
# The following packages are considered to be unsafe in a requirements file:
setuptools==42.0.2        # via cairocffi, plaster, pyramid, tinycss2, zope.deprecation, zope.interface, zope.sqlalchemy

```